lib/web_ids.js

```diff
@@ -1,43 +1,43 @@
-<<<<<<< HEAD
+// read environment variables from .env file
+require('dotenv').config();
+
 const redis = require('redis');
 const fs = require('fs');
-const {promisify} = require('util');
 const path = require('path');
-=======
-const redis = require("redis");
-const fs = require("fs");
 
 // global vars that store signatures
-ipSignature = [];
-headerSignature = [];
-bodySignature = [];
-statusSignature = [];
+let IP_SIGNATURES = [];
+let HEADER_SIGNATURES = [];
+let BODY_SIGNATURES = [];
+let STATUS_SIGNATURES = [];
 
 /**
  * Creates client and connects to the redis with default port settings
  * Loads all json files containing signatures for pattern matching and detection
  * and * attaches an event listener to redis client to analyze new requests
  * */
 function init() {
-  const subscriber = redis.createClient();
+  const seconds = 0;
+  const messageCount = 0;
+  loadSignatures(path.join(__dirname, '..', 'signatures'));
 
-  let seconds = 0;
-  let messageCount = 0;
-  loadSignatures("../signatures/");
+  if (process.env.NODE_ENV === 'production') {
+    const subscriber = redis.createClient();
 
-  subscriber.on("message", function (channel, message) {
-    messageCount += 1;
+    subscriber.on('message', (_channel, message) => {
+      messageCount += 1;
 
-    const formatted_message = JSON.parse(message);
+      const formattedMessage = JSON.parse(message);
 
-    console.log('\n');
-    console.log(`Request #${messageCount}:`);
-    console.log(formatted_message);
+      console.log('\n');
+      console.log(`Request #${messageCount}:`);
+      console.log(formattedMessage);
```

Comment:
why leaving console logs in production environment?

Reply:
Should I completely hide it in production environment?

Reply:
in production we will log to a file, so i would put it outside of the production context and put it into an development environment.

Reply:
Sounds nice to me, I'll adjust it accordingly!